src/tests/ftest/util/dmg_utils.py

```diff
@@ -1135,7 +1135,7 @@ def config_generate(self, access_points, num_engines=None, scm_only=False,
             use_tmpfs_scm (bool, optional): Whether to use a ramdisk instead of PMem
                 as SCM. Defaults to False.
             control_metadata_path (str): External directory provided to store control
```

Comment:
```suggestion
            control_metadata_path (str, optional): External directory provided to store control
```

Reply:
apologies I failed to update this in the most recent push, I will apply it if I have to re-push